src/inmanta/agent/agent_new.py

```diff
@@ -22,6 +22,7 @@
 from concurrent.futures.thread import ThreadPoolExecutor
 from typing import Any, Optional
 
+import inmanta.protocol  # noqa: F401 prevent import loop
```

Comment:
What's this?

Reply:
it prevents an import loop (nasty one we already have, but we always happen to load protocol before inmanta.data.model)

Reply:
Ah no, you didn't fix it, just worked around it here?

Reply:
I also can't seem to reproduce it. Can you tell me more?

Reply:
I managed to remove two already

Reply:
```
conftest.py:31: in <module>
    from inmanta.agent.handler import CRUDHandler, HandlerContext, ResourceHandler, SkipResource, TResource, provider
../src/inmanta/agent/__init__.py:19: in <module>
    from inmanta.agent.agent_new import Agent
../src/inmanta/agent/agent_new.py:26: in <module>
    from inmanta import config, const, data, protocol
../src/inmanta/data/__init__.py:55: in <module>
    from inmanta.data import model as m
../src/inmanta/data/model.py:34: in <module>
    import inmanta.protocol
../src/inmanta/protocol/__init__.py:56: in <module>
    from . import methods, methods_v2
../src/inmanta/protocol/methods.py:30: in <module>
    from inmanta.protocol.common import ArgOption
../src/inmanta/protocol/common.py:46: in <module>
    from inmanta.data.model import BaseModel, DateTimeNormalizerModel
E   ImportError: cannot import name 'BaseModel' from partially initialized module 'inmanta.data.model' (most likely due to a circular import) (/home/wouter/projects/inmanta-core/src/inmanta/data/model.py)
```